pkg/ext-proc/handlers/response.go

```diff
@@ -73,6 +73,7 @@ func (s *Server) HandleResponseBody(reqCtx *RequestContext, req *extProcPb.Proce
 		return nil, fmt.Errorf("unmarshaling response body: %v", err)
 	}
 	reqCtx.Response = res
+	reqCtx.ResponseSize = len(body.ResponseBody.Body)
```

Comment:
@liu-cong what will we get here for streaming requests?

Reply:
@JeffLuoo can we pls document in the metrics README doc the semantics of this metric as @liu-cong described?

Reply:
Updated the README.